README.md

```diff
@@ -0,0 +1,41 @@
+# SelfID
+
+Monorepo for the SelfID [application](#application) and [SDK](#sdk).
+
+## Installation
+
+This monorepo uses Yarn workspaces, make sure to install it first if you don't already have it.
+
+1. `yarn install` to install the dependencies
+1. `yarn build` to build all the packages
+
+### Additional scripts
+
+- `yarn lint` to run the linter in all packages
+- `yarn test` to run tests in all packages
+- `yarn docs` to generate API documentation
+
+## Application
+
+The SelfID application is available using Ceramic's mainnet on [self.id](https://self.id) and the Clay testnet on [clay.self.id](https://clay.self.id).
+
+## SDK
+
+### [Documentation](https://developers.ceramic.network/tools/self-id/overview/#sdk)
+
+### Packages
+
+| Name                                             | Description                                                                                                                 | Version                                                               |
```

Comment:
I really like this, should consider adding it to more of our monorepos.

Reply:
Thanks! Yes I find it quite convenient usually.